src/axom/sidre/core/Group.hpp

```diff
@@ -1587,10 +1602,15 @@ class Group
    *
    * Note: This is for the "sidre_{zzz}" protocols.
    *
+   * \param export_buffer  Optional parameter, if set to false, the data
```

Comment:
To match the parameter name:
```suggestion
   * \param export_buffers  Optional parameter, if set to false, the data
```